utils/update_checkout/update-checkout-config.json

```diff
@@ -173,6 +176,7 @@
                 "swift": "release/6.0",
                 "cmark": "release/6.0",
                 "llbuild": "release/6.0",
+                "swift-toolchain-sqlite": "release/6.0",
```

Comment:
I'll defer to @shahmishal if it makes sense to include these in the release branch schemes

Reply:
I think that we should branch these as otherwise we will not be able to reproduce the builds when we update the repository.